src/aics-image-viewer/shared/constants.ts

```diff
@@ -43,8 +42,7 @@ export const
   DENSITY_SLIDER_LEVEL_DEFAULT = [50],
   LEVELS_SLIDER_DEFAULT: ColorArray = [35.0, 140.0, 255.0],
   OTHER_CHANNEL_KEY = "Other",
-  SINGLE_GROUP_CHANNEL_KEY = "Channels",
```

Comment:
was SINGLE_GROUP_CHANNEL_KEY completely unused?

Reply:
It's still there, git just logged it as a deletion + addition because it has a semicolon after it now rather than a comma.